### PR TITLE
feat: Add DeparturesNoData component to v2 eink

### PR DESCRIPTION
**Pull request by cmaddox5**
**Asana task**: [[E-ink] Add DeparturesNoData component to eInk front-end apps](https://app.asana.com/0/1185117109217413/1202131702864644/f)

Based on the [these designs](https://share.goabstract.com/5961689f-1714-4a37-bcfc-bee55d876aa7?collectionLayerId=42bf6e24-c328-4146-95ec-e1d579e765d1&mode=design). I needed to add to the GlEink template to accommodate a top body takeover.

- [ ] Tests added?

### CODE REVIEW

File: assets/css/v2/eink/departures_no_data.scss
Comment (jzimbel-mbta) [inline assets/css/v2/eink/departures_no_data.scss:31]: Hmm, I thought for e-ink we only used black/white/grayscale. Could you check with Mindy that this is definitely the color we want?

File: assets/src/components/v2/eink/departures_no_data.tsx
Comment (jzimbel-mbta) [inline assets/src/components/v2/eink/departures_no_data.tsx:9]: Same Q about allowable e-ink colors here

**Review by jzimbel-mbta — APPROVED** [review]
Approved assuming the non-grayscale colors are ok!

File: assets/src/components/v2/eink/departures_no_data.tsx
Comment (cmaddox5) [reply assets/src/components/v2/eink/departures_no_data.tsx]: Went back and double-checked the designs and this needs to be solid black `#000000`. Got that changed.

File: assets/css/v2/eink/departures_no_data.scss
Comment (cmaddox5) [reply assets/css/v2/eink/departures_no_data.scss:31]: I'll double-check with design before merging. E-Ink NoData state also uses this same color for the divider, so I'm thinking it's ok. But they might also both be incorrect.

Comment (cmaddox5) [reply assets/css/v2/eink/departures_no_data.scss]: Great callout on this. Checked with Mindy and she agreed that these should be solid black. Went ahead and did this for `NoData` as well.

**Pull request merged**
